test cases/linuxlike/5 dependency versions/meson.build

```diff
@@ -50,6 +50,11 @@ assert(somelib_reqfalse.found(), 'somelib should have been found')
 somelibver = dependency('somelib',
   version : '>= 0.3',
   fallback : ['somelibver', 'some_dep'])
+# Find an internal dependency again with impossible multi-version
+somelibver = dependency('somelib',
+  version : ['>= 0.3', '<0.3'],
+  required : false,
+  fallback : ['somelibver', 'some_dep'])
```

Comment:
Shouldn't this assert  that the returned object is not found?

Reply:
It could, indeed. The reason of that test is because without the patch that would raise backtrace. PR updated with the assert.